pypose/lietensor/basics.py

```diff
@@ -41,6 +41,40 @@ def vec2skew(input:torch.Tensor) -> torch.Tensor:
                         torch.stack([-v[...,1],  v[...,0],         O], dim=-1)], dim=-2)
 
 
+def skew2vec(input:torch.Tensor) -> torch.Tensor:
+    r"""
+    Convert batched skew matrices to vectors.
+
+    Args:
+        input (Tensor): the skew matrices :math:`\mathbf{x}` to convert.
+
+    Return:
+        Tensor: the tensor :math:`\mathbf{y}`.
+
+    Shape:
+        Input: :obj:`(*, 3, 3)`
+
+        Output: :obj:`(*, 3)`
+
+    .. math::
+        {\displaystyle \mathbf{y}_i={\begin{bmatrix}\!-x_{i,2,3}
+        \\\,\,\,x_{i,1,2}\\\!-x_{i,1, 1}\end{bmatrix}}}
+
+    Note:
+        The last 2 dimension of the input tensor has to be (3, 3).
+
+    Example:
+        >>> pp.skew2vec(torch.randn(1, 3, 3))
+        tensor([[-0.2929, -1.2761, 2.2059]])
+    """
+    v = input.tensor() if hasattr(input, 'ltype') else input
+    assert v.shape[-2:] == (3, 3), "Last 2 dim should be (3, 3)"
+    # assert torch.equal(v.permute(0, 2, 1), -v), "Each matrix must be a skew matrix"
```

Comment:
This line is commented because with using cuda devices, when calculating the matrix C with torch.mm(A, B.T) - torch.mm(B, A.T), the result would not even meet with C == -C.T. But when use CPU to perform such operations, the result would meet with C == -C.T.